### PR TITLE
add doc for slow store detection

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the document about slow store detection.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s):
  - https://github.com/tikv/tikv/pull/10674
  - https://github.com/tikv/tikv/pull/10656
  - https://github.com/tikv/pd/pull/3922

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -638,6 +638,13 @@ raftstore 相关的配置项。
 + 默认值：1
 + 最小值：大于 0
 
+### `inspect-interval`
+
++ 定期检测 Raftstore 的延迟情况，并且当检测延迟超过该时间时会被记为超时。
```
Comment (qiancai) [inline tikv-configuration-file.md:643]: 请问定期是什么样的频率呢？这里是否可以说明呢

**Review by rleungx — APPROVED** [review]

File: tikv-configuration-file.md (hunk 1 above)
Comment (5kbpers) [reply tikv-configuration-file.md:643]: `inspect-interval` 可以设置间隔时间，这里的描述需要修改吗

**Review by disksing — APPROVED** [review]

File: best-practices/pd-scheduling-best-practices.md [hunk 2]
```diff
@@ -280,3 +280,5 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
 没有人工介入时，PD 处理 TiKV 节点故障的默认行为是，等待半小时之后（可通过 `max-store-down-time` 配置调整），将此节点设置为 Down 状态，并开始为涉及到的 Region 补充副本。
 
 实践中，如果能确定这个节点的故障是不可恢复的，可以立即做下线处理，这样 PD 能尽快补齐副本，降低数据丢失的风险。与之相对，如果确定这个节点是能恢复的，但可能半小时之内来不及，则可以把 `max-store-down-time` 临时调整为比较大的值，这样能避免超时之后产生不必要的副本补充，造成资源浪费。
+
+在 5.2.0 中引入了 TiKV 的慢节点检测机制，通过对 TiKV 中的请求进行采样计算出一个范围在 1~100 的分数。当某个分数大于 80 时，该节点会被设置为 Slow 状态。可以通过添加 `evict-slow-store-scheduler` 来针对慢节点进行对应的检测和调度，目前仅支持当且仅当出现一个慢节点，将慢节点上的 leader 全部驱逐（其作用类似于 `evict-leader-scheduler`）。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:284]: 当前文档里没有 evict-slow-store-scheduler 的说明，是否需要补充？
```suggestion
自 v5.2.0 起，TiKV 引入了慢节点检测机制。通过对 TiKV 中的请求进行采样，计算出一个范围在 1~100 的分数。当分数大于 80 时，该节点会被设置为 Slow 状态。可以通过添加 `evict-slow-store-scheduler` 来针对慢节点进行对应的检测和调度，目前支持当且仅当出现一个慢节点时，将慢节点上的 leader 全部驱逐（其作用类似于 `evict-leader-scheduler`）。
```

File: tikv-configuration-file.md (hunk 1 above)
Comment (TomShawn) [reply tikv-configuration-file.md:643]:
```suggestion
+ TiKV 每隔一段时间会检测 Raftstore 的延迟情况，该配置项设置检测的时间间隔。当延迟检测的间隔超过该时间时，会被记为超时。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**